examples/integration/glyphs/selection_nonselection.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+from bokeh.io import save
+from bokeh.plotting import figure
+
+p = figure(x_range=(0, 6), y_range=(0, 7), width=600, height=600, toolbar_location=None)
+
+def mkglyph(x, y, selection):
+    r = p.circle(
+        x, y,
+        color='green',
+        size=40,
+        selection_color='red',
+        selection_alpha=0.2,
+        nonselection_color='black')
+    r.data_source.selected['1d'] = dict(indices=selection)
```

Comment:
This will not work after @clairetang6 selection PR, it would need to use the new `Selection` model

Reply:
That PR isn't merged yet and I don't expect this PR being merged after.

Reply:
I'm saying that you should put a link in the other PR with a note to make sure that updating it gets done.